packages/block-library/src/navigation/index.php

```diff
@@ -132,6 +132,84 @@ function block_core_navigation_render_submenu_icon() {
 	return '<svg xmlns="http://www.w3.org/2000/svg" width="12" height="12" viewBox="0 0 12 12" fill="none" role="img" aria-hidden="true" focusable="false"><path d="M1.50002 4L6.00002 8L10.5 4" stroke-width="1.5"></path></svg>';
 }
 
+
+/**
+ * Finds the first non-empty `wp_navigation` Post.
+ *
+ * @return WP_Post|null the first non-empty Navigation or null.
+ */
+function block_core_navigation_get_non_empty_navigation() {
+	$navigation_posts = get_posts(
+		array(
+			'post_type' => 'wp_navigation',
+		)
+	);
+
```

Comment:
What is the default ordering here? I mean, the first from what list? ID?

Reply:
Good point. It uses `[WP_Query](https://developer.wordpress.org/reference/classes/wp_query/#order-orderby-parameters)` so by default it's:

 - order `DESC`
 - orderby `date (post_date)`

We should probably look to emulate the same type of ordering as `[wp_get_nav_menus()](https://github.com/WordPress/wordpress-develop/blob/ba943e113d3b31b121f77a2d30aebe14b047c69d/src/wp-includes/nav-menu.php#L613-L632)` which defers to `get_terms()`.

Can't narrow down the order/order-by args as yet.

Reply:
Ok `orderby` is set to `name`. Why that is useful I don't know...

Reply:
https://github.com/WordPress/gutenberg/pull/36740/commits/2831ff35cf6373a75fac2a983e0e7b1a8ba35f23